README.md

```diff
@@ -46,12 +46,20 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
+* Install Docker Community Edition and docker-compose
```

Comment:
Oh; I didn't realize this was an outer list. Could they be headers, instead? Headers establish an implicit outline ordering in HTML pages. I missed the `*` (I'm used to lists being indented with two spaces).